packages/cadl-ranch-specs/http/azure/client-generator-core/internal/main.tsp

```diff
@@ -32,30 +36,56 @@ Expected response body:
 }
 ```
 """)
-@route("/getInternal")
+@route("/public")
 @get
-@global.Azure.ClientGenerator.Core.internal
-op getInternal(@query name: string): InternalModel;
+op publicOnly(@query name: string): PublicModel;
 
 @scenario
 @scenarioDoc("""
-This test is testing an internal operation using a non-internal model. The model is only used in this internal operation. The operation and model should be generated but not exposed.
-Expected body:
-```json
-{
-  "id": 1,
-  "name": <any string>
-}
-```
+This is an internal operation. It should be generated but not exposed.
+Expected query parameter: name=<any string>
 Expected response body:
 ```json
 {
-  "id": 1,
   "name": <any string>
 }
 ```
 """)
-@route("/postInternal")
-@post
+@route("/internal")
+@get
 @global.Azure.ClientGenerator.Core.internal
-op postInternal(@body body: ModelOnlyUsedByInternalOperation): ModelOnlyUsedByInternalOperation;
+op internalOnly(@query name: string): PublicModel;
+
+@route("/shared")
+interface Shared {
+  @scenario
```

Comment:
Put the scenario on the interface so both operations belong to the same scenario